rules/game_process.tex

```diff
@@ -178,33 +190,99 @@ \subsubsection{Indirect Kick - Champions Cup}
   \item[Example 6:] Player 2 (of the red team) is awarded a \texttt{Penalty kick} and kicks the ball towards the blue-team's goal. The goalkeeper blocks the ball. The referee then calls ``Ball Free``.
   Player 2 (of the red team) kicks the ball again and score.
   This is \textit{not} an indirect kick, the goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 7:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks directly into the blue goal.
+  The failed-free-kick exception applies, and the goal counts.
+  \item[Example 8:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks the ball 1 meter away.
+  The ball stops, Player 2 kicks again, and scores.
+  The exception does \textit{not} apply for the red team because Player 2 played the ball twice. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 9:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) passes the ball to Player 3 (of the red team), who kicks again and scores.
+  Although the exception doesn't apply because Player 3 played the ball as well, this is still an ordinary indirect kick, so the goal counts.
+  \item[Example 10:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks the ball 1 meter away.
+  Player 5 (of the blue team) kicks the ball, then Player 2 kicks and scores.
+  The exception does \textit{not} apply for the red team because Player 5 also played the ball. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 11:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, then Player 5 (of the blue team) kicks the ball.
+  Player 2 (of the red team) then kicks the ball and scores.
+  The exception does \textit{not} apply for the red team because a blue-team robot played the ball first. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
 \end{description}
 
-\subsubsection{Indirect Kick - Challenge Shield}
-\label{sec:indirect_kick_challenger}
+\subsubsection{Fallback mode for Challenge Shield}
+
+This rule aims to allow \textbf{Challenge Shield} teams to play games
+even in the unfortunate case where severe circumstances make them unable to
```

Comment:
We should specify early the exact number of robot early in the text description for this to apply